examples/window/clear_color.rs

```diff
@@ -1,3 +1,5 @@
+//! Creates a solid color window.
```

Comment:
```suggestion
//! Shows how to set the solid color that is used to paint the window before the frame gets drawn.
//! 
//! Acts as background color, since pixels that are not drawn in a frame remain unchanged.
```